### PR TITLE
COL-3032: Updating SDK

### PR DESCRIPTION
**Pull request by vuryss**

### CODE REVIEW

File: src/Transfers/Constants.php [hunk 1]
```diff
@@ -207,7 +209,7 @@ class Constants
     const REASON_NO_REACTION_BUYER = 'NoReactionBuyer';
     const CARRIER_CODE_OTHER = 'Other';
     const CARRIER_CODE_OTHER_HAULER = 'Other Hauler';
-    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP = 'Amazon Logistics DE (Swiship)';
+    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP_ = 'Amazon Logistics DE (Swiship)';
```
Comment (ridaamirini) [inline src/Transfers/Constants.php]: NIT: Is there any reason, why we need this trailing dash?

```suggestion
    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP = 'Amazon Logistics DE (Swiship)';
```

**Comment by mansurs** [comment]
We need to update the CHANGELOG.md here too

File: src/Transfers/Constants.php (hunk 1 above)
Comment (vuryss) [reply src/Transfers/Constants.php]: Probably not but this was auto-generated so I didn't change it.

**Comment by vuryss** [comment]
> We need to update the CHANGELOG.md here too

Done

**Pull request merged**